types/documentdb/index.d.ts

```diff
@@ -126,12 +129,8 @@ interface AbstractMeta extends UniqueId {
 
 /** Represents a custom document for storing in DocumentDB  */
 interface NewDocument<TContent> extends UniqueId {
```

Comment:
Agreed, good catch. My goal was to change as little of the original definition while expanding coverage of the SDK. I might revisit the core resource definitions in a later update.

I've updated this PR to remove TContent.